beetmoverscript/src/beetmoverscript/templates/firefox_partner.yml

```diff
@@ -8,6 +8,8 @@ mapping:
   {{ 'en-US' }}:
     target.tar.bz2: firefox-{{ version }}.tar.bz2
     target.tar.bz2.asc: firefox-{{ version }}.tar.bz2.asc
+    target.tar.xz: firefox-{{ version }}.tar.xz
```

Comment:
I'm actually not sure if these are used anymore...but this won't hurt. You'll definitely need to update the in tree manifests (which I assume you already know.)